doc/whats_new.rst

```diff
@@ -38,13 +38,19 @@ NEW
   to enable annotation of a slice coordinate position with the float.
 
 
+
 Fixes
 -----
 
 - :class:`nilearn.input_data.NiftiLabelsMasker` no longer ignores its `mask_img`
 - :func:`nilearn.masking.compute_brain_mask` has replaced
   nilearn.masking.compute_gray_matter_mask. Features remained the same but
   some corrections regarding its description were made in the docstring.
+- the default background (MNI template) in plotting functions now has the
+  correct orientation; before left and right were inverted.
+- :func:`nilearn.glm.first_level.hemodynamic_models` can deal with regressors
```

Comment:
```suggestion
- first level modelling can deal with regressors
```
Indeed the hemodynamic_models.py is not a function which causes the CI to fail, nor a real module. Anyway I don't think we need to reference this precisely since its a bugfix no changes in API nor behavior